### DISCUSSION TITLE
世另我 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#485] [General]
# 世另我 - Miao Yu | 于淼

https://yufree.cn/cn/2024/12/21/another-yufree/

**Comment by hongtaoh** [comment]
你这篇是看了 Ilya Sutskever 的演讲后想到的吗？

>我们现在所处的2024年末可以电子化的人类知识基本都被拿去训练大语言模型了

我感觉还有不少内容没办法被大模型理解的，比如音乐、视频。还有不少私有的数据，大模型也没有权限。如果单纯拿文字来说的话，也许如此。但数据以及其他媒体载体的，大模型目前还没有吃进去。

>给他们眼睛与手去探索

完全同意。我认为这个工作需要由智能机器人来完成。想象一下，大街上都是机器人，观察世界、更新世界模型 (Large World Model)。有一些机器人被人类购买，它们可以近距离观察人类，更了解人类社会的运作机制以及人类的情感。

>而且手机的传感器是开放给语言模型的，这其实就给了新高质量数据收集的机会

我倒是没有想过借人类的手来扩建世界模型。这么一想的话，特斯拉估计也在干这件事。这样一来，隐私泄密的风险就大大增加了。


你想做的 YuGPT，诗涛做过类似的：https://shitao5.org/chat/

>我是不介意世界上出现另一个我的，但世界也许很介意。

为啥世界要介意？

**Reply by yufree** [reply]
我是看了演讲的报道，也认同优质文本资料到头了。不过多模态能提供多少附加信息我是存疑的，从信息量上视频<音频<文字，人行为上存在太多同质化，单纯堆数据估计资源都耗在去除噪音上了。推理模型外接传感器这条路可能比继续从人这边堆数据更有效，要让机器从观察阶段进入实验阶段才能发现更丰富新知识，当然这些知识是不是对人有用就另当别论了。
诗涛这个是走的RAG路线，我这个属于基础模型微调，技术路线与应用场景不太一样，我看好RAG做的本地知识库，但风格化输出可能微调更适合。
世界介意是因为有伦理问题。假设A在未取得作家B同意下用B公开作品数据训练了一个数字版B，然后用数字版B出版B写的作品续作，此时B是否被A侵权？还是说被自己的数字版B侵权？如果我们把数字版B换成偷了细胞做的克隆人呢？另一个自己会不会对自己造成伤害？本体与克隆体是平均享受福利还是本体优先？有克隆体跟没克隆体的人投票权是否一致？克隆体能否继承本体遗产？这些问题都还不是当前世界伦理规范可调控的，但技术上确实可以部分实现。这种问题不上称都不会介意，上了称就变成再怎么介意也不为过了，我说我不介意是因为我当前还意识不到这个数字版对其他人的高风险，说世界介意是因为在我想不到的别人视角里风险会很高。

**Reply by hongtaoh** [reply]
同意你说的视频信息量比较少。如果单就信息量来说，那现在确实已有文字信息基本上都喂给大模型了。

>诗涛这个是走的RAG路线，我这个属于基础模型微调，技术路线与应用场景不太一样，我看好RAG做的本地知识库，但风格化输出可能微调更适合。

了解。

你提到的伦理问题让我想到美国之前的[一次判决](https://www.fromgeek.com/ai/598028.html)裁定 AI 生成的艺术品不构成侵权。大模型用了很多受版权保护的作品用于训练，现在貌似也没有明确说这构成侵权。
